plugins/rapid7_intsights/icon_rapid7_intsights/util/api.py

```diff
@@ -138,6 +138,13 @@ def get_alerts(self, alert_params: AlertParams) -> list:
     def add_manual_alert(self, manual_alert_params: ManualAlertParams) -> str:
         return self.make_request("PUT", "public/v1/data/alerts/add-alert", json_data=manual_alert_params.to_dict()).text
 
+    def get_cve(self, cve_ids: [str]) -> dict:
+        path = "public/v1/cves/get-cves-list"
+        if cve_ids:
+            cve_ids_str = "&cveId[]=".join(cve_ids)
+            path = f"{path}?cveId[]={cve_ids_str}"
+        return self.make_json_request("GET", path)
```

Comment:
What is correct behavior here if the user doesn't pass any cve list at all? To me it seems like we aren't adding any params to the url

Reply:
If user not pass any cve list there will be no parameter in request and the API return all cve.